requirements/ci.txt

```diff
@@ -4,6 +4,10 @@
 #
 #    make upgrade
 #
+--index-url http://edx.devstack.devpi:3141/root/pypi/+simple/
```

Comment:
🙌

Reply:
Bah, this was the thing that made every test run take 15+ minutes, so I yanked these out.